python/common/sql-versioning/sql_versioning/versioning.py

```diff
@@ -61,19 +82,20 @@ def _is_session_modified(session):
     return False
 
 
-def _get_operation_type(session, obj):
+def _get_operation_type(session, obj, delete_orphan: False):
```

Comment:
LGTM 😸 just a very tiny thing here
```suggestion
def _get_operation_type(session, obj, delete_orphan=False):
```

Reply:
Ah good catch on this typo! Fixed